src/contexts/TodosContext.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { Todo } from '../types/Todo';
+import { TodoActions } from '../types/TodoActions';
+import { useLocalStorage } from '../hooks/UseLocalStorage';
+
+type Props = {
+  todos: Todo[];
+  setTodos: (todosToSet: Todo[]) => void;
+  selectedFilter: TodoActions;
+  setSelectedFilter: (action: TodoActions) => void;
+  isShownTodos: Todo[]
+};
+
+export const TodosContext = React.createContext<Props>({
+  todos: [],
+  setTodos: () => {},
+  selectedFilter: TodoActions.All,
+  setSelectedFilter: () => {},
+  isShownTodos: [],
+});
+
+// eslint-disable-next-line
+export const TodoProvider = ({ children }: { children: React.ReactNode }) => {
+  const [todos, setTodos] = useLocalStorage('todos', []);
+  // eslint-disable-next-line
```

Comment:
why did you disable eslint here? Don't do it w/o good reason - try to figure out with the problem instead

Reply:
not fixed

Reply:
but it's mean about line 22. that I was fixed.